tracing-attributes/src/lib.rs

```diff
@@ -153,6 +153,14 @@ mod expand;
 ///     // ...
 /// }
 /// ```
+/// Overriding the generated span's parent:
+/// ```
+/// # use tracing_attributes::instrument;
+/// #[instrument(parent = None)]
+/// pub fn my_function() {
+///     // ...
+/// }
+/// ```
```

Comment:
nit, take it or leave it: it might also be nice to have an example where the parent is not `None`...perhaps a method on a struct with a span in it?
```suggestion
/// # use tracing_attributes::instrument;
/// #[instrument(parent = None)]
/// pub fn my_function() {
///     // ...
/// }
/// ```
///
/// ```
/// # use tracing::instrument;
/// // A struct which owns a span handle.
/// struct MyStruct {
///     span: tracing::Span,
///     // ...
/// }
///
/// impl MyStruct {
///     // Use the struct's `span` field as the parent span
///     #[instrument(parent = &self.span)]
///     fn do_stuff() {
///         // ...
///     }
/// }
```

Reply:
Addressed by https://github.com/tokio-rs/tracing/pull/2091/commits/cf1d59e23c8d907df1b33d66d7b06c61280cc4ba